### ISSUE TITLE
"ChatGPT" - not stated on website anymore - Copy buttons not available

### ISSUE THREAD

**Issue by jsjs19xx** [Foxxey/CopyEquation#6]
Hi, thanks for the nice feature!

i think because the layout of chatGPT changed the copy buttons are missing:

![image](https://github.com/Foxxey/CopyEquation/assets/131341487/02a6ca36-0a5d-4406-89fa-c66f0486a5bd)

Can you please check? :)

Edit: I saw that a similar issue was opened. Maybe some html / css element name changed? Because for me it is not working.

**Comment by gilgaTheDev** [comment]
No problem for me with the current layout of ChatGPT.
![image](https://github.com/user-attachments/assets/4dea4ce0-b837-46ee-89be-3a1f418ca827)

**Comment by jsjs19xx** [comment]
@gilgaTheDev yes this is the context menu for one formula (if you right click on it) that works for me too.

But I'm searching for those buttons:
![image](https://github.com/user-attachments/assets/c8f584df-c1b8-4009-84b2-6f45d92fe19b)

...

**Comment by jsjs19xx** [comment]
@Foxxey ist this a general problem?

**Comment by ka161heg** [comment]
Is there any fix for copying the whole message and not only individual formulas?

Note: Thank you for this amazing extension btw.

**Comment by UtkarshIEEE** [comment]
Everything was working well until a few days back. Not sure if an update changed things or what, but now I only see the box when hovering over equations. When I click on it, I don't get the copy options.
